README.md

```diff
@@ -37,14 +37,41 @@ dotnet run --project TKeazirian.HTTPServer
 
 This project uses xUnit.net for testing. Visit [this][5] website for more information and how to use xUnit.net.
 
+This project also uses Acceptance tests written in Cucumber Code found [here][9]. You can read more about Cucumber Code [here][10].
+
 ### To Run the Tests:
 
 ```shell
 dotnet test
 ```
 
+### Acceptance Tests:
+**Setup**
+
+1. You will need [Ruby][6] version `2.5.1`.  You can install Ruby using [brew][7] or the Ruby [docs][8]. 
+2. Once it is installed, run `ruby -v` to confirm you are using the appropriate version.
+3. Next, run `bundler install`.
+
+**Steps to Run**
```

Comment:
When making a choice between [emphasis vs headers](https://cirosantilli.com/markdown-style-guide/#emphasis-vs-headers) in markdown, choose headers.

```suggestion
#### Steps to Run
```